Source/WebCore/platform/graphics/MediaSourcePrivate.h

```diff
@@ -69,13 +69,12 @@ class MediaSourcePrivate : public RefCounted<MediaSourcePrivate> {
     virtual void waitForSeekCompleted() = 0;
     virtual void seekCompleted() = 0;
 
-    virtual void setTimeFudgeFactor(const MediaTime& fudgeFactor) { m_timeFudgeFactor = fudgeFactor; }
-
-    MediaTime timeFudgeFactor() const { return m_timeFudgeFactor; }
     bool isSeeking() const { return m_isSeeking; }
 
+    WEBCORE_EXPORT static const MediaTime& currentTimeFudgeFactor();
+    WEBCORE_EXPORT static bool hasFutureTime(const MediaTime& currentTime, const MediaTime& duration, const PlatformTimeRanges&);
+
```

Comment:
I would rather not make the "currentTimeFudgeFactor()" static global. I suspect we will have to pull this back into non-static scope in the relatively near term if the MSE v2 spec settles on a solution for allowing this value to be specified by the page.

Reply:
ok. Will keep it then.